agent/acl.go

```diff
@@ -104,6 +104,12 @@ func newACLManager(config *Config) (*aclManager, error) {
 					Policy: acl.PolicyWrite,
 				},
 			},
+			Nodes: []*acl.NodePolicy{
+				&acl.NodePolicy{
+					Name:   "",
```

Comment:
Is the name logged anywhere? if so would it make sense to give it a friendlier name than ""?

Otherwise LGTM